src/Core/System/SalesChannel/Event/SwitchContextEvent.php

```diff
@@ -53,4 +53,12 @@ public function getParameters(): array
     {
         return $this->parameters;
     }
+
+    /**
+     * @param array<string, mixed> $parameters
+     */
+    public function setParameters(array $parameters): void
```

Comment:
I am not sure, if overwriting the whole parameters is okay :thinking: what about an "addParameter" method, that takes a string key and a mixed value?

or do you also need to remove stuff?

Reply:
For me an add would be enough. I'll change it and add a changelog file.